### PR TITLE
add file upload validation

### PR DESCRIPTION
**Pull request by will-byrne**
Closes libero/reviewer#1093

### CODE REVIEW

File: src/initial-submission/components/FileDetailsForm.tsx [hunk 1]
```diff
@@ -42,14 +46,38 @@ const FileDetailsForm = ({ initialValues, schemaFactory, ButtonComponent }: Step
     });
     const schema = schemaFactory(t);
 
+    const validationResolver = useCallback((data: FileDetails) => {
+        try {
+            schema.validateSync(
+                { ...data, manuscriptFile: manuscriptStatus.fileStored.fileName },
```
Comment (Cooryd) [inline src/initial-submission/components/FileDetailsForm.tsx:52]: Could write a test for this. Load page with initialValues not having manuscriptFile then try with one and see if it validates correctly?

Comment (Cooryd) [inline src/initial-submission/components/FileDetailsForm.tsx:52]: Does the current schema check the manuscriptFile property already?

Comment (will-byrne) [reply src/initial-submission/components/FileDetailsForm.tsx:52]: I added it to the existing schema

Comment (Cooryd) [reply src/initial-submission/components/FileDetailsForm.tsx:52]: I must have missed that on my firstpass. Looks good

**Pull request closed without merging**